src/middleware.js

```diff
@@ -240,11 +253,16 @@ function buildParametersValidation(parameters) {
         delete data.in;
         delete data.required;
 
-        if (required) {
-            destination.required = destination.required || [];
-            destination.required.push(key);
+        if (data.type === 'file') {
+            // destination.items.enum.push(key);
```

Comment:
Is this commented out code needed?

Reply:
old code from my first solution before the custom keyword - deleted